app/server/utils/user.py

```diff
@@ -871,10 +873,25 @@ def attach_transfer_account_to_user(user, organisation=None):
         transfer_account = TransferAccount(bound_entity=user,
                                            organisation=organisation,
                                            blockchain_address=blockchain_address)
-
         db.session.add(transfer_account)
-
         user.default_transfer_account = transfer_account
+
+        org_users = organisation.users
+        initial_disbursement = config.SELF_SERVICE_WALLET_INITIAL_DISBURSEMENT
+        disbursing_admin_list = []
+        for org_user in org_users:
+            if org_user.has_admin_role:
```

Comment:
im not sure I understand what is being done here. But in any case you could break on first, because you only use the first one below here anyway.

Reply:
Forgive my ignorance, didn't know I could do that, let me find out how

Reply:
`break` :)